framework/src/main/java/org/checkerframework/common/basetype/BaseTypeValidator.java

```diff
@@ -701,4 +703,53 @@ public boolean areBoundsValid(
 
         return true;
     }
+    /**
+     * Validate if the annotations on wildcard are permitted by @TargetLocations. Report an error if
+     * the actual use of this annotation is not listed in the declared TypeUseLocations
+     * in @TargetLocation.
+     *
+     * @param type the type to check
+     * @param tree the tree of this type
+     */
+    protected void validateWildCardTargetLocation(AnnotatedWildcardType type, Tree tree) {
+        if (visitor.ignoreTargetLocation) return;
```

Comment:
change the access modifier of `ignoreTargetLocation` to public? or adding a getter method. As in `inferenceValidator`, we need to know the value of this flag.